package.json

```diff
@@ -1,21 +1,49 @@
 {
-  "name": "pug-monorepo",
-  "private": true,
+  "name": "madthax-pug-ts",
```

Comment:
This is just the monorepo config. It is not a package, so it needs to remain private so we don't publish it by accident.

Also, npm will show the packages you published regardless of the organization they are in ([check mine](https://www.npmjs.com/~svallory)) so how about we name this `@tokilabs/pug-ts` (after the repository) and all other packages as `@tokilabs/[original name]`?